analyses/models.py

```diff
@@ -154,6 +160,23 @@ class StudyFeatures(BaseModel):
     def __str__(self):
         return self.accession
 
+    def set_results_dir_default(self):
```

Comment:
This feels like it probably belongs elsewhere, though exactly where depends on what odd case we're trying to catch? Should it be a migration? Or a management command we run after certain bad things happen? Or somewhere in the flows?